test/fuzz/README.md

```diff
@@ -96,7 +96,7 @@ directory. Fuzzing continues indefinitely until a bug is found or the number of
 perform is specified with `-runs`. For example,
 
 `bazel run //test/common/common:base64_fuzz_test --config asan-fuzzer
```

Comment:
can i suggest putting this in a code block - ie with 

```markdown
```console
...
```
i think the command needs a `\` at the end also to make it work multiline in bash

Reply:
sure, let me update, thanks for the review!